ramls/schema/circulation-rule-match.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Loan rule match schema",
+  "title": "Circulation rule match schema",
   "type": "object",
   "properties": {
-    "loanRuleLine": {
+    "circulationRuleLine": {
```

Comment:
similarly, this could be `ruleLine`?

Reply:
Changed in https://github.com/folio-org/mod-circulation/commit/4b0a55b49ecc627393e69877032f53a9554e7d7f